business_requirement_deliverable/models/business.py

```diff
@@ -82,7 +82,7 @@ def _compute_currency_id(self):
             if currency_id:
                 brd.currency_id = currency_id
             else:
-                brd.currency_id = self.env.user.company_id.currency_id
+                brd.currency_id = self.env.company.currency_id
```

Comment:
This might actually change behavior, right?

- `self.env.user.company_id` would be the default company associated with the user (and this field still exists in v14
- `self.env.company` would be the main company activated in the environment

Can this change in behavior be a problem?

Reply:
In my opinion it is better to use `self.env.company`.
What do you think your @pedrobaeza ?

Reply:
Isn't there another company available right there in the model? If not, the change seems good for me and in fact it should be changed in v13, not here (but no need to be backported).

Reply:
I have changed it to use the value of `business_requirement_id.currency_id` from the record.

Reply:
Ok, looks good :+1: